py3status/modules/getjson.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Query a url and display the json response
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds
+        (default 30)
+    format: placeholders will be replaced by the returned json key names
+        (default '')
+    sep: the separator between parent and child objects
+        (default '_')
+    timeout: how long before deciding we're offline
+        (default 5)
+    url: specify a url to fetch json from
+        (default '')
+
+Format placeholders:
+    Placeholders will be replaced by the json keys
+    Placeholders for objects with sub-objects are flattened using 'sep' in between
+    (eg. {'parent': {'child': 'value'}} will use placeholder {parent_child}
+
+@author vicyap
+"""
+
+import collections
+import json
+try:
+    # python3
+    from urllib.request import urlopen
+except:
+    from urllib2 import urlopen
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 30
+    format = ''
+    sep = '_'
+    timeout = 5
+    url = ''
+
+    def _flatten(self, d, parent_key=None):
+        """
+        modified from http://stackoverflow.com/a/6027615
+        """
+        items = []
+        for k, v in d.iteritems():
```

Comment:
`{}.iteritems()` does not work with python3 use `{}.items()` instead as it will work for both 2 and 3.